collector/minio/collector.yaml

```diff
@@ -0,0 +1,34 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'minio-job'
+          scrape_interval: 5s
+          # MinIO deployments started with MINIO_PROMETHEUS_AUTH_TYPE set to "public" can omit the bearer_token field.
```

Comment:
Just for the record I like how this is done. Still minimal, but it blends in a little extra potentially valuable information.